pkg/cliplugins/workload/plugin/sync.go

```diff
@@ -416,11 +451,15 @@ func (o *SyncOptions) getResourcesForPermission(ctx context.Context, config *res
 			return false, nil //nolint:nilerr
 		}
 
+		if len(syncTarget.Spec.SupportedAPIExports) == 0 {
+			return true, nil
+		}
+
 		// skip if there is only the local kubernetes APIExport in the synctarget workspace, since we may not get syncedResources yet.
 		clusterName := logicalcluster.From(syncTarget)
 
 		if len(syncTarget.Spec.SupportedAPIExports) == 1 &&
-			syncTarget.Spec.SupportedAPIExports[0].Export == "kubernetes" &&
+			syncTarget.Spec.SupportedAPIExports[0].Export == workloadv1alpha1.ImportedAPISExportName &&
 			(len(syncTarget.Spec.SupportedAPIExports[0].Path) == 0 ||
 				syncTarget.Spec.SupportedAPIExports[0].Path == clusterName.String()) {
```

Comment:
This test mainly doesn't work anymore (after the workspace refactoring), especially if the local export has been added with the path, and not the cluster name.
Don't we want to ensure that `imported-apis` (or any other name) would be a reserved name, and then we wouldn't even allow adding it as an `apiexports` arg, especially if it has a path ?

Reply:
the check here is to get the synced resources list from the sycnTarget, so we can correctly set the rbac for syncer on spoke. However, if there are only imported-apis APIExport, we can not get this list because the list is only updated when syncer is started.

Reply:
Yes, I know this.

I was speaking of this part of the test:

`syncTarget.Spec.SupportedAPIExports[0].Path == clusterName.String()`